replicatedhq/ship
# Add HelmValues lifecycle step and save endpoint

What I Did
------------
- Add `HelmValues` lifecycle step
- Add `helm-values` endpoint

How I Did it
------------
- Add `HelmValues` to render persisted `values.yaml` from provided github url
- Add `helm-values` endpoint to save edited values to `state`

How to verify it
------------
- Run `kustomize` subcommand with a valid helm chart url
- POST to `helm-values` endpoint with a valid chart `values.yaml` in the following format:
```
  {
    "values": ...
  }
```
- If a 200 response code is received, verify that the `state.json` has been written with the `helmValues`

Description for the Changelog
------------
Add HelmValues lifecycle step and helm-values endpoint

Picture of a Boat (not required but encouraged)
------------
![](http://vignette2.wikia.nocookie.net/gundam/images/c/c6/Irish-klasse.jpg/revision/latest?cb=20130403232151)
